src/containerapp-preview/azext_containerapp_preview/_params.py

```diff
@@ -0,0 +1,21 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+from azure.cli.core.commands.parameters import get_enum_type
+
+from ._utils import (_get_or_add_extension, _get_azext_module, GA_CONTAINERAPP_EXTENSION_NAME)
+from ._validators import validate_env_name_or_id
+
+
+def load_arguments(self, _):
+    if not _get_or_add_extension(self, GA_CONTAINERAPP_EXTENSION_NAME):
+        return
+    azext_params = _get_azext_module(
+        GA_CONTAINERAPP_EXTENSION_NAME, "azext_containerapp._params")
+    azext_params.load_arguments(self, _)
```

Comment:
We'd better ensure that this extension can run independently.

Reply:
Containerapp-preview extension rely on containerapp. Just like aks-preview relay on acs in az-cli command_modules.
We need to develop preview feature in existed command.
We not recommand containerapp-preview duplicate all GA code.